### PR TITLE
[Resolves #35] Add links to headings

### LINKED ISSUE #35: Generate anchor tags in headings rendered by marked
Like so: https://github.com/chjj/marked#overriding-renderer-methods

### PR DESCRIPTION
**Pull request by Vijar**
@jedireza @redonkulus

### CODE REVIEW

File: utils/renderer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    return '<h' + level + '>' +
+                '<a name="' +
+                escapedText +
+                 '" class="anchor"></a>' +
+                text +
+                 ' <a href="#' +
+                 escapedText +
+                 '" class="hash-link">#</a>' +
+                '</h' + level + '>';
```
Comment (jedireza) [inline utils/renderer.js]: Maybe we could format this differently. Here is an option:
```js
    return '<h' + level + '>' +
        '<a name="' + escapedText + '" class="anchor"></a>' +
        text +
        ' <a href="#' + escapedText + '" class="hash-link">#</a>' +
    '</h' + level + '>';
```

**Comment by jedireza** [comment]
LGTM :rocket:

File: utils/renderer.js (hunk 1 above)
Comment (redonkulus) [inline utils/renderer.js]: +1

**Comment by redonkulus** [comment]
:shipit:

**Pull request merged**